chore(doc): add troubleshooting doc for virtual disk blockdevices (#30)
openebs/upgrade: docs/migration.md

- [SPC pools to CSPC pools](#spc-pools-to-cspc-pools)
- [cStor External Provisioned volumes to cStor CSI volumes](#cstor-external-provisioned-volumes-to-cstor-csi-volumes)

**Note:** 
 - If the Kubernetes cluster is on rancher and iscsi is running inside the kubelet container then it is mandatory to install iscsi service on the nodes and add extra binds to the kubelet container as mentioned [here](https://docs.openebs.io/docs/next/prerequisites.html#rancher).
 - Minimum version of Kubernetes to migrate to CSPC pools / CSI volumes is 1.17.0.

>  - If using virtual disks for blockdevices in cStor please refer this [doc](virtual-disk-troubleshoot.md) before proceeding.

Reviewer: How about the below contents?
```suggestion
 - If using virtual disks as blockdevices for provisioning cStorpool please refer this [doc](virtual-disk-troubleshoot.md) before proceeding.
```
